h/zssLogging.h

```diff
@@ -518,8 +518,10 @@ bool isLogLevelValid(int level);
 #ifndef ZSS_LOG_JWK_FAILED_MSG_ID
 #define ZSS_LOG_JWK_FAILED_MSG_ID     ZSS_LOG_MSG_PRFX"1605W"
 #endif
-#define ZSS_LOG_JWK_FAILED_MSG_TEXT   "Server will not accept JWT\n"
+#define ZSS_LOG_JWK_FAILED_MSG_TEXT   "Server will not accept JWT\nCheck URL https://%s:%d%s for errors.\n"
```

Comment:
I suggest we have a single messages: `Server will not accept JWT\nCheck URL %s for errors.\n` and format the path in a separate function. The reason for that is for a simple task (print an error message) we're spilling so much unnecessary details here and in `jwkTaskMain`; it's better to isolate that.